test/NuGet.Frameworks.Test/FrameworkReducerTests.cs

```diff
@@ -11,6 +12,74 @@ namespace NuGet.Test
     public class FrameworkReducerTests
     {
         [Theory]
+        // Generation fall-through patterns of uap10.0: 
+        // 1. lib/uap10, 
```

Comment:
nit: `uap10.0` (I know the source of this comment had `uap10` but that's `1.0` :P)